regexes/device/mobiles.yml

```diff
@@ -2716,6 +2741,17 @@ Mediacom:
     - regex: 'M-MP([0-9a-z]+)'
       model: 'SmartPad $1'
 
+# MediaTek
+MediaTek:
+  regex: 'MT[68]\d{3}[_);/ ]'
+  models:
+    - regex: '(MT8\d{3})'
+      model: '$1'
+      device: 'tablet'
+    - regex: '(MT\d{4})'
+      model: '$1'
+      device: 'smartphone'
+
```

Comment:
Already thought about adding MediaTek a few times before. But I'm still unsure if those are real devices. For me it seems MediaTek is a vendor for mobile chipsets. See http://www.mediatek.com/en/products/mobile-communications/smartphone1/